backend/src/yeomanui.ts

```diff
@@ -13,6 +13,9 @@ import { IRpc } from "@sap-devx/webview-rpc/out.ext/rpc-common";
 import Generator = require("yeoman-generator");
 import { GeneratorType, GeneratorFilter } from "./filter";
 
+const uniq = require('array-uniq');
+const win32 = process.platform === 'win32';
```

Comment:
it is a boolean, but from the name it is very hard to know.
Change it to something like: isWin32